emails/decommission_profile/fr/html.ejs

```diff
@@ -0,0 +1,10 @@
+<p>Cher/chère <%= user.name %>,</p>
+
+<p>Merci d'être membre de Humanitarian ID.</p>
+<p>Nous vous informons que les fonctionnalités de gestion des contacts de HID seront mises hors service en janvier 2021, conformément à son cadre technologique <a href="https://blog.angular.io/stable-angularjs-and-long-term-support-7e077635ee9c">en raison de la fin de vie de sa base technologique</a>. Veuillez noter que vous pouvez continuer à utiliser la plateforme jusqu'à cette date, mais nous vous recommandons <a href="https://docs.google.com/document/d/14x-wL1Yr0Il4gKgVuMxlP3GnaVyanS-QFhgUcCe1M50/edit?usp=sharing">d'exporter tous les contacts et listes</a> pertinents à l'avance.</p>
```

Comment:
@rupl Sorry, the following should be amended:

```
conformément à son cadre technologique <a href="https://blog.angular.io/stable-angularjs-and-long-term-support-7e077635ee9c">en raison de la fin de vie de sa base technologique</a>.
```

to 

```
<a href="https://blog.angular.io/stable-angularjs-and-long-term-support-7e077635ee9c">en raison de la fin de vie de sa base technologique</a>.
```

Reply:
New PR here: https://github.com/UN-OCHA/hid_api/pull/88